packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -105,6 +104,7 @@ import { shouldMakeEgg } from "../resources";
 import { lavaDogsAccessible, lavaDogsComplete } from "../resources/doghouse";
 import { hotTubAvailable } from "../resources/clanVIP";
 import { meatMood } from "../mood";
+import { getAvailableUltraRareZones } from "garbo-lib/dist/wanderer/lib";
```

Comment:
this isn't quite how we want to import this; make sure the function is exported from `garbo-lib`'s index, then run `yarn`, then import it just from `garbo-lib`